.github/workflows/mobilecoin-workflow-dev-setup-environment.yaml

```diff
@@ -219,6 +223,15 @@ jobs:
         object_name: consensus-minting-secrets
         src: ${{ env.MINTING_BASE_PATH }}
 
+  setup-environment-v5-0:
```

Comment:
5.0 charts rely on fog-services and fog-services-config. The new charts don't use fog-services and fog-services-config.

Take different paths for 5.0 restore and new 5.1 charts and undo some of the abstraction. This makes the workflow longer, but maybe easier to follow all the environment setup steps.